Makefile

```diff
@@ -69,6 +69,7 @@ test: $(SOURCE) VERSION .gobuild
 	@echo Testing inside Docker container for $(GOOS)/$(GOARCH)
 	docker run \
 	    --rm \
+	    -t \
```

Comment:
@Nesurion IIRC you had issues with `-ti`. I added `-t` to be able to abort tests running in containers with ctrl+c. Do you remember if there is an issue with this?